internal/reconcilers/entity_add.go

```diff
@@ -52,21 +51,17 @@ func (r *Reconciler) handleEntityAddEvent(msg *message.Message) error {
 		return nil
 	}
 
-	var repoOwner string
-	var repoName string
+	var entName string
 	var ok bool
-	if repoOwner, ok = event.Entity["repoOwner"].(string); !ok {
-		return errors.New("invalid repo owner")
-	}
-	if repoName, ok = event.Entity["repoName"].(string); !ok {
-		return errors.New("invalid repo name")
+	if entName, ok = event.Entity[properties.PropertyName].(string); !ok {
+		return fmt.Errorf("invalid or unset entity name: %s", entName)
```

Comment:
Will this continue using the name? Or will this switch to using the upstream ID?

Reply:
interesting idea, this code goes to repositories.CreateRepository which uses the name as a property for RetrieveAllProperties. I'm not sure if that and especially the underlying code supports retrieving a repo by name. I need to check that.

Reply:
It's not a big deal what property is used, as long as we use properties and not tie something to names.